sdk/web-pubsub/web-pubsub/package.json

```diff
@@ -118,5 +97,31 @@
     "skip": [
       "managingGroups.js"
     ]
+  },
+  "type": "module",
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "selfLink": false
+  },
+  "browser": "./dist/browser/index.js",
```

Comment:
Should this package have a browser dialect so that we have build output to dist/browser?

Reply:
No, it shouldn't.  Removing